src/mocks/handlers.js

```diff
@@ -133,7 +133,7 @@ Object.keys(groupedResponses).forEach((endpoint) => {
       status = response;
       response = undefined;
     } else if (/post|patch|put|delete/.test(method)) {
-      status = response ? 201 : 204;
+      status = response ? 200 : 204;
```

Comment:
I think 201 was ok. Which case did you have in mind where we want 200?

Reply:
Browse Datasets page did not work because the querySearchService and getUserIVAs expect a 200 otherwise there is an error

Reply:
The getUserIVAs uses a GET request, so this would not be handled here.

Regarding the search service, you're right, it expects a 200 from a POST request.

We can cover it like this then
```ts
status = response ? (response.hits ? 200 : 201) : 204;
```

Reply:
That works yeah